frontend/test/metabase/scenarios/collections/permissions.cy.spec.js

```diff
@@ -285,6 +297,43 @@ describe("collection permissions", () => {
                   cy.location("pathname").should("eq", "/collection/root");
                   cy.findByText("Orders").should("not.exist");
                 });
+
+                describe("adding question to dashboard", () => {
+                  beforeEach(() => {
+                    cy.skipOn(user === "nodata");
```

Comment:
Why do we skip every test for "nodata" user?

Reply:
Ouch, haven't noticed it. Removing, thanks!